Makefile

```
@@ -226,4 +228,4 @@ dangerous-drop-all-dbs:
 		psql -h localhost -U postgres -c "DROP DATABASE \"$$db\""; \
 	done
 
-ci: lint typecheck check-dependencies test
+ci: lint typecheck lint-dependencies test
```

checkmake (0.3.2)

[warning] Target "ci" should be declared PHONY.

(phonydeclared)